src/Card/CardWithImage.styles.js

```diff
@@ -22,35 +11,46 @@ const imageContain = (value, idx) => {
   return value[idx < value.length - 1 ? idx : value.length - 1]
 }
 
+const borderRadius = vertical => ({
+  borderRadius: vertical ? '8px 8px 2px 2px' : '2px 8px 8px 2px'
+})
+
 export default {
   container: ({ theme, vertical }) => ({
     overflow: 'hidden',
     ...borderRadius(true),
     ...sm(theme)(borderRadius(vertical))
   }),
 
-  contentBox: ({ theme, vertical, ratio }) => ({
-    ...dimensions(true, ratio),
-    ...sm(theme)(dimensions(vertical, ratio))
+  contentBox: ({ theme, ratio }) => ({
+    flex: `${1 + _extract(ratio, 0, 0)} 0 ${100 * _extract(ratio, 0, 0)}%`,
+    ...sm(theme)({
+      flex: `${1 + _extract(ratio, 1, 0)} 0 ${100 * _extract(ratio, 1, 0)}%`
+    }),
+    ...md(theme)({
+      flex: `${1 + _extract(ratio, 2, 0)} 0 ${100 * _extract(ratio, 2, 0)}%`
+    }),
+    ...lg(theme)({
+      flex: `${1 + _extract(ratio, 3, 0)} 0 ${100 * _extract(ratio, 3, 0)}%`
+    })
```

Comment:
@alsofronie 👍 sa verifici si in Safari sa vezi daca merge cum trebuie proprietatea asta de flex, ca astia is mai dubiosi
https://stackoverflow.com/questions/45802179/safari-flex-grow-behavior-different-from-chrome-ff-edge-css-flexbox

Reply:
<img src="https://media.giphy.com/media/YBHJyPCU9h1VewdaPZ/giphy.gif" />